test/qa/test_meshnet.py

```diff
@@ -277,6 +277,19 @@ def check_rules_routing() -> (bool, str):
         assert result, message
 
 
+def test_connect_set_mesh_off():
+    output = f"{sh.nordvpn.mesh.peer.list(_tty_out=False)}"
+    peer = meshnet.get_peers(output)[0]
+    sh.nordvpn.mesh.peer.connect(peer)
+    sh.nordvpn.disconnect()
+    sh.nordvpn.connect()
+    sh.nordvpn.set.mesh.off()
+
+    with lib.Defer(sh.nordvpn.set.mesh.on):
```

Comment:
why is needed to have meshnet `on` at the end of the test?

Reply:
To bring daemon back to the same state as after `setup_module`.